app/assets/javascripts/components/overview/salesforce_link.jsx

```diff
@@ -16,6 +16,12 @@ const SalesforceLink = createReactClass({
     ServerActions.linkToSalesforce(this.props.course.id, salesforceId);
   },
 
+  updateSalesforceId() {
```

Comment:
Unrelated change.

Reply:
ugh! Mixed it up with salesforce id branch. Will remove.